web3/web3/method.py

```diff
@@ -0,0 +1,127 @@
+import utils.utils
+import errors
+
+
+class Method:
```

Comment:
Writing classes like this comes from an older python that didn't have `object`.

The current *one-way-to-do-things* in python regarding classes is to inherit from `object` (unless you have a compelling reason not to).  Mind updating this?